### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by yahyaelganyni1**
- Add splash screen
- Add login and signup screens
- Add pages to view categories and purchases
- Add pages to add new categories and purchases
- Add tests

Video presentation: [view](https://www.loom.com/share/df221dd6c3da48229be020f64baf0ebe)

### CODE REVIEW

**Comment by arslanbisharat** [comment]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
- [x] Make sure you have added all unnecessary files in gitignore.

- [x] Please tell us what you liked in the project in your project video.


- [x] Tests should be properly added. Make sure you have more tests added. 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by yahyaelganyni1** [comment]
Video presentation : [here](https://www.loom.com/share/df221dd6c3da48229be020f64baf0ebe)

File: .gitignore [hunk 1]
```diff
@@ -29,3 +29,10 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+/public/packs
+/public/packs-test
+/node_modules
```
Comment (rloterh) [inline .gitignore:35]: - [x] `node_modules` directory is still showing in the project structure of your repository. Kindly ensure to properly remove this directory by clearing the GitHub cache in your local repository

image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Yahya, 

You did great on this project, and for making the changes requested in the previous review(s)! 👏👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- No linter errors

## Required Changes ♻️
_Kindly check the comments below this review. Kindly take your time to carefully improve aspects outlined in the previous code review(s), and please check to see that the changes are working as intended._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 3 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
Hi @yahyaelganyni1, 

Great work on suggestions.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**